namespaces/live-1.cloud-platform.service.justice.gov.uk/court-probation-dev/resources/versions.tf

```diff
@@ -1,4 +1,4 @@
 
 terraform {
-  required_version = "0.12.17"
+  required_version = ">= 0.12"
```

Comment:
@jhevans , we are using terraform version "0.12.17", across the cluster. 

We want to maintain same version across all the namespaces as the pipeline is using "0.12.17" to apply terraform.

Reply:
We received an advisory to upgrade from the pen testers so I figured if I applied this we would get any future patch upgrades. If that's the case you're right there's no point in applying these changes. Will close